sdk/documentintelligence/Azure.AI.DocumentIntelligence/src/AnalyzeResult.Serialization.cs

```diff
@@ -1,19 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-namespace Azure.AI.DocumentIntelligence
-{
-    public partial class AnalyzeResult
-    {
-        // AnalyzeResult.FromResponse was not included in the generated code, but it should be.
-        // This is just a workaround while this issue is not fixed or while we don't implement
-        // deserialization manually.
-
-        /// <summary> Deserializes the model from a raw response. </summary>
-        /// <param name="response"> The response to deserialize the model from. </param>
-        internal static AnalyzeResult FromResponse(Response response)
-        {
-            return DeserializationHelpers.FromOperationResponse(response, DeserializeAnalyzeResult, "analyzeResult");
```

Comment:
It seems that the operation deserialization issue has been fixed. The generator now creates a private helper method in the client to parse the result of an operation.

Reply:
I'm not touching other `FromResponse` methods I added manually since they're still being used by the non-generated layer of the code. We'll eventually get rid of those as other code gen problems get fixed.